src/PQSelectionPanel.cxx

```diff
@@ -121,10 +181,9 @@ void PQSelectionPanel::constructor()
 void PQSelectionPanel::clicSelection(QTreeWidgetItem* item, int column)
 {
 	unsigned int cpt;

```

Comment:
I think this variable is not used. Please remove it.
It is a warning on Visual Studio.